Kerdy/Kerdy/MyProfile/ProfileTagBtn.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ProfileTagBtn.swift
+//  Kerdy
+//
+//  Created by 최다경 on 12/27/23.
+//
+
+import UIKit
+
+class ProfileTagBtn: UIStackView {
+    
+    var label: UILabel = {
+        let label = UILabel()
+        label.font = .nanumSquare(to: .regular, size: 13)
+        return label
+    }()
+    
+    var closeBtn = UIButton()
+    
+    let img: UIImageView = {
+        let img = UIImageView()
+        img.image = .icCancel
+        return img
+    }()
+    
+    init() {
+        super.init(frame: .zero)
+        
+        self.axis = .horizontal
+        self.axis = .horizontal
+        self.alignment = .fill
+        self.distribution = .equalSpacing
+        self.spacing = 0
+        
+        closeBtn.addTarget(self, action: #selector(closeBtnTapped), for: .touchUpInside)
+        setLayout()
+    }
+    
+    private func setLayout() {
+        self.addArrangedSubviews(label)
+        self.addArrangedSubviews(closeBtn)
```

Comment:
```suggestion
        self.addArrangedSubviews(label, closeBtn)
```
stackView의 extension에 구현한 addArrangedSubviews를 사용하신다면 위와 같이 사용하는 건 어떨까요?